build.gradle

```diff
@@ -2626,6 +2638,12 @@ project(":controls") {
 
 project(":swing") {
 
+    // We need to skip setting compiler.options.release for this module,
+    // since javafx.swing requires jdk.unsupoprted.desktop, which is
```

Comment:
jdk.unsupoprted.desktop

typo !

Reply:
Fixed. I had the same typo in the CSR (copy/paste) so I fixed it there, too.

Thanks.